src/i18n/de.json

```
@@ -415,6 +415,9 @@
         "success": "Dein Vorschlag wurde erfolgreich abgesendet. Vielen Dank noch einmal! Wir werden ihn jetzt schnellstmöglich überprüfen.",
         "error": "Beim Absenden deines Vorschlags ist leider ein Fehler aufgetreten. Bitte stelle sicher, dass Du mit dem Internet verbunden bist und Anfragen an externe Server nicht mit einem Add-On wie uMatrix oder NoScript blockierst.",
         "no-input": "Bitte gib etwas ein, um Deinen Vorschlag abzusenden.",
+        "name-or-web-missing": "Bitte gib mindestens einen Namen oder eine Webseite an.",
+        "invalid-request": "Deine Eingaben entsprechen nicht dem richtigen Format oder sind unvollständig. Korrigiere sie und schicke den Vorschlag erneut ab.",
+        "github-error": "Beim Absenden gab es ein Problem mit GitHub. Dein Vorschlag wurde nicht angenommen. Versuch es doch noch einmal.",
```

'angenommen' sounds like a human decision that the record doesn't meet our standards. I think the 'registered' you used in the English translation is way more clear. Maybe 'abgeschickt'? While not technically true, it more accurately explains the problem to the user imo.

```suggestion
        "github-error": "Beim Absenden gab es ein Problem mit GitHub. Dein Vorschlag wurde nicht abgeschickt. Versuch es doch noch einmal.",
```